Dockerfile

```diff
@@ -22,12 +34,19 @@ COPY pkg/ pkg/
 ARG VERSION=devel
 ARG TARGETARCH
 ARG TARGETOS
-RUN --mount=type=cache,target=/root/.cache/go-build CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "-X github.com/piraeusdatastore/drbd-shutdown-guard/pkg/vars.Version=$VERSION" -o drbd-shutdown-guard main.go
+RUN --mount=type=cache,target=/root/.cache/go-build \
+    CGO_ENABLED=0 \
+    GOOS=$TARGETOS \
+    GOARCH=$TARGETARCH \
+    go build -a -ldflags "-X github.com/piraeusdatastore/drbd-shutdown-guard/pkg/vars.Version=$VERSION" -o drbd-shutdown-guard main.go
 
-FROM --platform=$TARGETPLATFORM registry.access.redhat.com/ubi9/ubi:latest
+# === Stage 3: Final minimal runtime image ===
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
```

Comment:
If you switch to ubi-minimal, you also need to install `systemctl`, as the go binary requires it. So this change was completely untested and breaks the build.